packages/react-instantsearch-theme-algolia/style.scss

```diff
@@ -3,111 +3,105 @@
 // Functions
 // ------------------------------------
 @function str-replace($string, $search, $replace: '') {
-  $index: str-index($string, $search);
-
-  @if $index {
-    @return str-slice($string, 1, $index - 1) + $replace + str-replace(str-slice($string, $index + str-length($search)), $search, $replace);
-  }
-
-  @return $string;
+    $index: str-index($string, $search);
+    @if $index {
+        @return str-slice($string, 1, $index - 1)+$replace+str-replace(str-slice($string, $index + str-length($search)), $search, $replace);
+    }
+    @return $string;
 }
 
 // ------------------------------------
 // Mixins
 // ------------------------------------
-
 @mixin closedCaret() {
-  &:after {
-    content: '';
-    display: block;
-    position: absolute;
-    top: 50%;
-    right: -22px;
-    width: 14px;
-    height: 14px;
-    background: url("data:image/svg+xml;utf8,<svg viewBox=\'0 0 8 13\' xmlns=\'http://www.w3.org/2000/svg\'><path d=\'M1.5 1.5l5 4.98-5 5.02\' stroke=\'%23697782\' stroke-width=\'1.5\' fill=\'none\' fill-rule=\'evenodd\' stroke-linecap=\'round\' opacity=\'.4\'/></svg>")no-repeat center center / contain;
-    transform: translateY(-50%)
-  }
+    &:after {
+        content: '';
+        display: block;
+        position: absolute;
+        top: 50%;
+        right: -22px;
+        width: 14px;
+        height: 14px;
+        background: url("data:image/svg+xml;utf8,<svg viewBox=\'0 0 8 13\' xmlns=\'http://www.w3.org/2000/svg\'><path d=\'M1.5 1.5l5 4.98-5 5.02\' stroke=\'%23697782\' stroke-width=\'1.5\' fill=\'none\' fill-rule=\'evenodd\' stroke-linecap=\'round\' opacity=\'.4\'/></svg>")no-repeat center center / contain;
+        transform: translateY(-50%)
+    }
 }
 
 @mixin openCaret() {
-  &:after {
-    top: 25%;
-    transform: rotate(90deg)
-  }
+    &:after {
```

Comment:
I'm not sure if these style changes were on purpose. Could you do it in another PR if they're on purpose so that the diff is more readable?